build.gradle

```diff
@@ -259,7 +259,7 @@ dependencies {
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.13.2.1'
     implementation group: 'ch.qos.logback', name: 'logback-core', version: logbackVersion
     implementation group: 'ch.qos.logback', name: 'logback-classic', version: logbackVersion
-    implementation group: 'com.github.hmcts', name: 'ecm-common', version: '0.1.182'
+    implementation group: 'com.github.hmcts', name: 'ecm-common', version: '0.1.183'
```

Comment:
ecm-common latest is 0.1.185" just for consistency.

Reply:
updated